extensions/3DTILES_bounding_volume_S2/0.0.0/README.md

```diff
@@ -1,75 +1,92 @@
 # 3DTILES_bounding_volume_S2
 
+<!-- omit in toc -->
 ## Contributors
 
 - Sam Suhag, Cesium
 - Sean Lilley, Cesium
 - Peter Gagliardi, Cesium
 
+<!-- omit in toc -->
 ## Status
 
 Draft
 
+<!-- omit in toc -->
 ## Dependencies
 
-Written against 3D Tiles 1.0. It may be used in conjunction with [`3DTILES_implicit_tiling`](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/extensions/3DTILES_implicit_tiling/0.0.0).
+Written against 3D Tiles 1.0.
 
+<!-- omit in toc -->
 ## Optional vs. Required
 
 This extension is required, meaning it must be placed in both the `extensionsUsed` and `extensionsRequired` lists in the tileset JSON.
 
+<!-- omit in toc -->
 ## Contents
 
 - [3DTILES_bounding_volume_S2](#3dtiles_bounding_volume_s2)
-  - [Contributors](#contributors)
-  - [Status](#status)
-  - [Dependencies](#dependencies)
-  - [Optional vs. Required](#optional-vs-required)
-  - [Contents](#contents)
   - [Overview](#overview)
   - [Coordinate System](#coordinate-system)
-  - [Cell Token](#cell-token)
-  - [Heights](#heights)
+  - [Bounding Volume](#bounding-volume)
+    - [Cell Token](#cell-token)
+    - [Bounding Heights](#bounding-heights)
   - [Subdivision](#subdivision)
+  - [Property Reference](#property-reference)
   - [Schema Changes](#schema-changes)
 
 ## Overview
 
-[S2](http://s2geometry.io/) is a spherical geometry library that represents all data on a 3D sphere, unlike traditional libraries that use 2D planar projections. This makes it possible to represent the globe with no seams or singularities, with low distortion everywhere on Earth. The S2 library involves projecting the 6 faces of a cube on a unit sphere, creating 6 root "cells" that subdivide evenly into 4 tiles, into a quadtree structure.
+[S2](http://s2geometry.io/) is a spherical geometry library that maps the 6 faces of a unit cube onto the unit sphere. Typically, traditional GIS libraries use planar projections to map data between a 2D plane and a 3D ellipsoid (representing the Earth). Since a sphere is a closer approximation of the shape of the ellipsoid, S2 makes it possible to represent the globe with no seams or singularities, with low distortion everywhere on Earth.
 
-This extension to 3D Tiles enables using S2 cells as a `boundingVolume`, and uses `3DTILES_implict_tiling` to enforce subdivision.
+|Level 0|Level 1|
+|:-:|:-:|
+| ![Plane - Level 0](figures/plane_root.png) S2 cell (`"1"`) on the Earth cube| ![Plane - Level 1](figures/plane_children.png) Children of S2 cell (`"1"`) on the Earth cube |
```

Comment:
Just a newbie S2 question: why are the 4 children `04, 0c, 14, 1c`? I would have thought they'd all start with 1 if this is face 1, but maybe S2 works differently than I think?